pkg/chart/dependency.go

```diff
@@ -53,6 +53,9 @@ type Dependency struct {
 // the chart. This check must be done at load time before the dependency's charts are
 // loaded.
 func (d *Dependency) Validate() error {
+	if d == nil {
+		return ValidationError("the dependency's item can not be nil")
```

Comment:
the dependency's item can not be nil --> dependency cannot be an empty list

Reply:
> the dependency's item can not be nil --> dependency cannot be an empty list

In  this case, dependency is not empty, it has one item *null*

Reply:
"dependency cannot be  empty"  may be better?

Reply:
There is no error as follows:

```yaml
dependency:
```

This causes the panic:

```yaml
dependencies:
-
```

That is why I think the message reads better as follows: "dependency cannot be an empty list"